pages/submission.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useSession, getSession, GetSessionParams } from 'next-auth/react';
+
+export default function Submission() {
+    const {data: session, status} = useSession();
+
+    if (status === "authenticated") {
```

Comment:
Is there a way to just use a single check to determine whether a user is signed in? We will be needing to do user authentication throughout the project, and having two if statements seem a little convoluted? I have not read the Next documentation, but maybe there is a better way of just using session or status